README.md

```diff
@@ -83,12 +102,18 @@ the following command to create the `dist` folder with required files.
 yarn run package
 ```
 
-## Pending Tasks
+## Pending Features
 
 - [ ] Typography support for iOS
+
 - [ ] Gradient colors support
+
 - [ ] Jetpack Compose support
 
+  
```

Comment:
Do we need to add the License here? I am not sure if we can add it now but maybe a line redirecting to the LICENSE file?

Reply:
Yeah, we can add it now as well